docs/plugins/artifactory.md

```diff
@@ -44,4 +44,15 @@ This plugin support`Ingress`, `ClusterIP`, `NodePort` and `LoadBalancer` , You c
 --8<-- "artifactory.yaml"
 ```
 
-Currently, except for `values_yaml`, all the parameters in the example above are mandatory.
+#### Default Configs
+
+| key | default value | description |
+|  ----  | ----  | ----  |
+| chart.chart_name | jfrog/artifactory | chart name |
+| chart.timeout | 10m | this config will wait 10 minutes to deploy |
+| upgradeCRDs | true | default update CRD config |
+| chart.wait | true | whether to wait until install is complete |
+| repo.url | https://charts.jfrog.io | offical helm repo address |
+| repo.name | jfrog | helm repo name |
+
+Currently, except for `values_yaml` and default configs, all the parameters in the example above are mandatory.
```

Comment:
I see this sentence, but there is no action in the code to set a default value to `values_yaml`.

It is also possible that I missed it.